pkg/rancher-desktop/config/__tests__/settings.spec.ts

```diff
@@ -553,9 +553,6 @@ describe('settings', () => {
             Debian:   false,
             openSUSE: true,
           },
-          experimental: {
```

Comment:
It doesn't make much sense to delete this: people with old config files would still have it, we need to test that we migrate it correctly (by dropping the field).